tm2/pkg/bft/rpc/core/ready.go

```diff
@@ -0,0 +1,54 @@
+package core
+
+import (
+	"fmt"
+
+	ctypes "github.com/gnolang/gno/tm2/pkg/bft/rpc/core/types"
+	rpctypes "github.com/gnolang/gno/tm2/pkg/bft/rpc/lib/types"
+)
+
+// Get node readyness. Returns 200 OK on success, 500 Internal Server Error - in
+// case of an error.
+//
+// ```shell
+// curl 'localhost:26657/ready'
+// ```
+//
+// ```go
+// client := client.NewHTTP("tcp://0.0.0.0:26657", "/websocket")
+// err := client.Start()
+//
+//	if err != nil {
+//	  // handle error
+//	}
+//
+// defer client.Stop()
+// result, err := client.Ready()
+// ```
+//
+// > The above command returns JSON structured like this:
+//
+// ```json
+//
+//	{
+//		"error": "",
+//		"result": {},
+//		"id": "",
+//		"jsonrpc": "2.0"
+//	}
+//
+// ```
+func Ready(ctx *rpctypes.Context) (*ctypes.ResultReady, error) {
+	var latestHeight int64
+	if getFastSync() {
+		latestHeight = blockStore.Height()
+	} else {
+		latestHeight = consensusState.GetLastHeight()
+	}
+
+	if latestHeight < 1 {
```

Comment:
Issue says
```
Readiness probe should respond in a positive way (200 http status code), when the network it is connected to is producing blocks, so latest block height is greater than 1.
Otherwise the endpoint should respond a non-200 http status code, with an optional payload with a reason.
```
So I think this should be  a '<=' sign ?
```suggestion
	if latestHeight <= 1 {
```

Reply:
`[ $(curl http://<RPC_ENDPOINT>/status | jq -r '.result.sync_info.latest_block_height') -ge 1 ]` was the ref call

Reply:
Nice thanks @sw360cab closing the comment then :)